docs/source/cc_basic.md

```diff
@@ -0,0 +1,209 @@
+# Running Chaincode as a Service on the Test Network
+
+The Fabric v2.4.1 chaincode-as-a-service feature is a novel, practical approach to running smart contracts. By comparison, the earlier method required the peer to orchestrate the complete lifecycle of the chaincode. This required the peer to have access to the Docker Daemon to create images, and to start containers. Java, Node.js and Go chaincode frameworks were explicitly known to the peer, including how they should be built and started.
+
+As a result, the earlier method made it challenging to deploy chaincode into Kubernetes (K8s) style environments, and to run chaincode in any form of debug mode. Additionally, the code was rebuilt by the peer, introducing a degree of uncertainty about which dependencies had been installed.
```

Comment:
..or other environments where access to the Docker Daemon is restricted. 

Would add 'usually rebuilt' as there are some specific cases where you can avoid this happening.

Reply:
Also - in the case of rebuilding it often needs internet access which may not always be available.

Reply:
ok.